### PR TITLE
Update cfn lint

### PR DESCRIPTION
**Pull request by paulopontesm**

### CODE REVIEW

**Comment by paulopontesm** [comment]
@lolgab can you please check if the tests are ok? I think there is something odd here

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,60 +1,56 @@
...
-            if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              docker login -u $DOCKER_USER -p $DOCKER_PASS
-              docker tag ${CIRCLE_PROJECT_REPONAME} codacy/${CIRCLE_PROJECT_REPONAME}:latest
-              docker tag ${CIRCLE_PROJECT_REPONAME} codacy/${CIRCLE_PROJECT_REPONAME}:1.0.$CIRCLE_BUILD_NUM
-              docker push codacy/${CIRCLE_PROJECT_REPONAME}:latest
-              docker push codacy/${CIRCLE_PROJECT_REPONAME}:1.0.$CIRCLE_BUILD_NUM
-            fi
-
+            docker build -t $CIRCLE_PROJECT_REPONAME:latest .
+            docker run $CIRCLE_PROJECT_REPONAME:latest codacy_prospector_test.py
```
Comment (ljmf00) [inline .circleci/config.yml]: Does this file exists? `codacy_prospector_test.py`

**Comment by ljmf00** [comment]
I added #9 to keep track of this PR.

File: .circleci/config.yml (hunk 1 above)
Comment (paulopontesm) [reply .circleci/config.yml]: it doesn't.. I copied this from a different project and didn't notice.
fixed

**Comment by paulopontesm** [comment]
@ljmf00 can you confirm me if this step is running as expected: 
https://circleci.com/gh/codacy/codacy-cfn-lint/17

Mainly the RunPatternTests. It doesn't log anything. Should I have a different structure on my tests?

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,60 +1,55 @@
...
+            - plugins_test
+          filters:
+            branches:
+              only:
+                - master
+      - codacy/tag_version:
+          name: tag_version
+          context: CodacyAWS
+          requires:
+            - publish_dockerhub
```
Comment (lolgab) [inline .circleci/config.yml]: Missing newline here

File: docs/patterns.json [hunk 3]
```diff
@@ -507,5 +617,5 @@
             "patternId": "E0002"
         }
     ],
-    "version": "0.8.3"
+    "version": "0.24.6"
 }
```
Comment (lolgab) [inline docs/patterns.json]: Missing newline here

**Review by lolgab — CHANGES_REQUESTED** [review]
The PR deletes all the test files. That's why there are no logs. Maybe the documentation generator empties the `docs` directory before adding the new files. So it deletes the `tests` directory.

**Comment by lolgab** [comment]
Changed the test files in the last commit. I updated cfn-lint again since a new version is out. @paulopontesm

**Review by lolgab — DISMISSED** [review]

**Review by lolgab — APPROVED** [review]

**Pull request merged**